### DISCUSSION TITLE
Which sensor to check if the router is connected?

### DISCUSSION THREAD

**Discussion by xeroski** [Vaskivskyi/ha-asusrouter#772] [General]
Hi.
Which sensor to check if the router is connected (online)?
I wanted to use "binary_sensor.{xxx}_wan" to check the status of the router (online or power supply). I know that it is used to check the WAN connection, and it works there, but I thought that if the HA is not connected to the router, most of the sensors will have the "unavailable" status. Unfortunately, the status is still "Connected"

**Comment by Vaskivskyi** [comment]
Hello. Sorry for the late reply. Please, let me check what will be the best solution for your case.
